fix missing cols function

awalsh272 opened the pull request:
Creates a fix_missing_cols function that checks for if all the columns in the "duplicate_subset" list exist, and if not instantiates a new column with all "None". 

Some minor tweaks as well. pd.np.nan --> np.nan

awalsh272 committed:
fix missing cols function

hopetambala commented on puente-data-exporter/lambdas/data-exporter/index.py:
```diff
@@ -121,4 +120,11 @@ def err_msg(msg: str) -> dict:
 
 if __name__ == '__main__':
-    handler()
+    event = {
+       "queryStringParameters": {
+            "surveyingOrganization": "Ryans Well",
+            "specifier": "HistoryEnvironmentalHealth"
+       } 
+    }
+    context = None
+    handler(event, context)
```
@awalsh272 we should change this back right?

awalsh272 replied:
Fixed in new commit

hopetambala commented on puente-data-exporter/lambdas/data-exporter/libs/utils.py:
```diff
@@ -17,6 +17,12 @@
     aws_secret_access_key=secretz.AWS_SECRET_ACCESS_KEY,
 )
 
+def fix_missing_cols(df, duplicate_subset):
+    missing_cols = [col for col in duplicate_subset if col not in df.columns]
+    for col in missing_cols:
+        df[col] = None
+    return df
```
@awalsh272 so just creates none values for data that doesn't exist?

awalsh272 committed:
main function ready for dashboard

To test index.py, you create an event and run the function. In the dashboard, the event is provided
